### ISSUE TITLE
Adapt NECB systems to use CCHP

### ISSUE THREAD

**Issue by khaddad** [canmet-energy/btap_tasks#176]
### Description
Update NECB systems to use heat pumps for heating and cooling. A heat pump will be used to condition the supply air to the zones and for local heating/cooling inside the zones.

### Testing Plan
Enter testing plan, if you are going to use an existing test or make a new one. 

### Waiting On
If this task is dependent on another task or NREL. Include notes Here. 

### Repositories Involved

https://github.com/canmet-energy/btap-development-environment

### Select the Project Code you will be charging to 2018-2019

| Client 	|WSBE  	|Name   	|Manager   	
|----------	|---	|---	|---	
|OEE    |P-001310.003.02.04 |TA OEE Support-1.4 CCHP for heating only|	Meli Stylianou

**Comment by khaddad** [comment]
An OS measure was developed to replace the heating and cooling coils of an air system with an air-source heat pump. The measure also replaces any zone baseboard with an air-source heat pump. This measure can then be applied to an NECB system and replace it with an air-source heat pump based system.

**Comment by phylroy** [comment]
@khaddad update this task to see if we should close.

**Comment by khaddad** [comment]
A second version of the CCHP measure was developed that deletes the existing HVAC system of an os model. Then an air system is specified with a CCHP with electric backup. Electric baseboard is also used. Results are generated with CCHP technology for the archetypes for 9 locations to show the energy, cost, and GHG performance of CCHP systems.
